app/controllers/v0/sessions_controller.rb

```diff
@@ -19,6 +19,12 @@ def new
       type = params[:signup] ? 'signup' : params[:type]
       if REDIRECT_URLS.include?(type)
         url = url_service.send("#{type}_url")
+
+        # If redirecting to ID.me, include GA clientId for cross-domain analytics
+        if %w[idme signup].include?(type)
+          url = params[:clientId] ? url + '&clientId=' + params[:clientId] : url
```

Comment:
Maybe use interpolation here?

```
clientId = params[:clientId]
url = clientId ? "#{url}&clientId=#{clientId}" : url
```

Reply:
I think this might be better here:  https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/saml/url_service.rb#L106

OneLogin::RubySaml::Authrequest.create takes params as a second argument.